server/dbconfig/seed_dev.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO posts (id, title, body)
```

Comment:
How would I ever access these posts? Is it only if I knew the long hash and put it into my URL?